pheweb/serve/server.py

```diff
@@ -223,10 +223,14 @@ def api_variant(query):
             die("Malformed variant query. Use chr-pos-ref-alt")
         v = Variant(q[0].replace('X', '23'),q[1],q[2], q[3])
         variantdat = jeeves.get_single_variant_data(v)
+        regions = jeeves.get_finemapped_regions(v)
```

Comment:
The regions should be moved back after the "if variantdat is None" branching, since there's no point in fetching the finemapped regions if the variant is not valid.